state/upgrades_test.go

```diff
@@ -3244,6 +3244,10 @@ func (s *upgradesSuite) TestAddControllerNodeDocs(c *gc.C) {
 		"_id":       ensureModelUUID(uuid1, "4"),
 		"machineid": "3",
 		"jobs":      []MachineJob{JobHostUnits},
+	}, bson.M{
```

Comment:
We just skip this machine because it doesn't have a novote key?

Reply:
yup